Sources/DesignSystem/Controllers/GDSScreen.swift

```
@@ -0,0 +1,189 @@
+import UIKit
+
+open class GDSScreen: BaseViewController, VoiceOverFocus {
+    public var initialVoiceOverView: UIView {
+        scrollViewInnerStackView.arrangedSubviews.first ?? UIView()
```

Rather than having a default UiView here, can we instead use a get throws and guard against the first ever being nil

---

Yeah it's something I brought over but didn't think much about. I guess `TitledViewController` doesn't really make sense if all the views will be passed in rather than constructed.

The change I made is to remove `TitledViewController` and made the change to VoiceOverFocus such that:
```swift 
public protocol VoiceOverFocus {
    var initialVoiceOverView: UIView { get throws }
}
```
the implementation then being:
```swift
public var initialVoiceOverView: UIView {
    get throws {
        guard let firstView = scrollViewInnerStackView.arrangedSubviews.first else {
            throw VoiceOverFocusError.notAvailable
        }
        return firstView
    }
}
```
the only thing here being that when this is called in BaseViewController the error is ignored as it's within a Task:
```swift
open override func viewIsAppearing(_ animated: Bool) {
    super.viewIsAppearing(animated)
    Task { @MainActor in
        if let screen = self as? VoiceOverFocus {
            UIAccessibility.post(notification: .screenChanged,
                                 argument: try screen.initialVoiceOverView)
        }
    }
}
```
but maybe that's what we want?